one/src/main.rs

```diff
@@ -81,27 +95,50 @@ struct DaemonOpts {
     #[arg(long, default_value_t = false, env = "CERAMIC_ONE_TRACING")]
     tracing: bool,
     /// Unique key used to find other Ceramic peers via the DHT
-    #[arg(
-        long,
-        default_value = "/ceramic/testnet-clay",
-        env = "CERAMIC_ONE_NETWORK_ID"
-    )]
-    network_id: String,
+    #[arg(long, default_value = "testnet-clay", env = "CERAMIC_ONE_NETWORK")]
+    network: Network,
+
+    /// Unique key used to find other Ceramic peers via the DHT
+    #[arg(long, env = "CERAMIC_ONE_LOCAL_NETWORK_ID")]
+    local_network_id: Option<u32>,
+
+    /// When true mdns will be used to discover peers.
+    #[arg(long, default_value_t = false, env = "CERAMIC_ONE_MDNS")]
```

Comment:
not sure what the best default value is here

Reply:
Mdns is not supported very well outside of LANs in home use. Generally datacenter/cloud networks do not allow mdns, as such its only valuable to the dev on their LAN if they are running multiple nodes. Since this seems like a narrow use case I decided default to off is best to reduce surface area of issues.